src/components/user/SignIn.js

```diff
@@ -1,65 +1,196 @@
-import { useRef, useState } from "react";
-import { signInWithEmailAndPassword } from "firebase/auth";
-import { auth } from '../../firebase';
-import { FaGoogle } from 'react-icons/fa';
-import { handleSignInWithPopup } from "../../functions/Auth";
+import React, { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import logoWithCircleBorder from '../../assets/logo_with_circle_border-removebg.png'; // Path to novelology logo in assets
 
 
-export function SignIn() {
-  const [loading, setLoading] = useState(false);
+// Define the SignIn component
+export const SignIn = (props) => {
 
-  const emailRef = useRef();
-  const passwordRef = useRef();
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
 
-  async function handleSignInWithEmailAndPassword() {
-    setLoading(true);
-    try {
-      await signInWithEmailAndPassword(auth, emailRef.current.value, passwordRef.current.value);
-    } catch (error) {
-      alert("Error signing in with email: " + error.message);
-    }
-    setLoading(false);
-  }
+  // Function to navigate to the registration page
+  const navigateToRegistration = () => {
+    window.location.href = '/register'; // URL of your registration page
+  };
+
+  // Main container styling with maroon/purple background color
+  const mainContainerStyle = {
+    width: '100%',
+    minHeight: '100vh',
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'center',
+    alignItems: 'center',
+    overflow: 'auto',
+    backgroundColor: 'rgba(91, 46, 72, 1)', // Background color for entire screen
+  };
+
+  const dontHaveAccountTextStyle = (fontSize, fontWeight, marginBottom, marginTop, color = 'rgba(255, 255, 255, 1)') => ({
+    color: color,
+    textAlign: 'center',
+    lineHeight: 'normal',
+    fontFamily: 'Montserrat',
+    fontSize: fontSize,
+    fontWeight: fontWeight,
+    textDecoration: 'none',
+    marginBottom: marginBottom,
+    marginTop: '60px', // Added marginTop for more control over spacing
+  });
+
+  // Adjusted sign in box styling for vertical coverage
+  const signInBoxStyle = {
+    maxWidth: '600px',
+    width: '100%',
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'space-around', // Ensures even distribution of space
+    alignItems: 'center',
+    position: 'relative',
+    overflow: 'hidden',
+    paddingTop: '20px', // Padding at the top
+    paddingBottom: '20px', // Padding at the bottom
```

Comment:
Unsure why we are using pixel specific styling again instead of reactive content that would allow for window resizing and mobile app possibility in the future